### DISCUSSION TITLE
File Format

### DISCUSSION THREAD

**Discussion by jmb-555** [sn4k3/UVtools#949] [General]
Since this seems to be one of the few groups of people on the planet who has figured out the "SVGX" format, I was wondering if anyone knew about the chunk of data at the beginning of the file. Most of the file can be decoded UTF-8 but there is a chunk of data at the beginning that shows up as gibberish. Does anyone know anything about this chunk?

**Comment by sn4k3** [comment]
As UVtools is open-source you can explore the file format to see how to read it.
Fast implementation can be seen [here](https://github.com/sn4k3/UVtools/blob/master/Scripts/010%20Editor/svgx.bt)
